resources/lang/de_DE/default.po

```diff
@@ -930,9 +901,6 @@ msgstr "Zusätzliche Beschreibung"
 msgid "This description is for single shifts, otherwise please use the description in shift type."
 msgstr "Diese Beschreibung ist für einzelne Schichten, ansonsten nutze bitte die Beschreibung im Schichttyp."
 
-msgid "Please select a location."
-msgstr "Bitte einen Ort auswählen."
```

Comment:
This is still used if  see that correctly

Reply:
but it was double 
its still there in line 296